bot/exts/evergreen/realpython.py

```diff
@@ -56,7 +56,7 @@ async def realpython(self, ctx: commands.Context, *, user_search: str) -> None:
         article_embed = Embed(
             title="Search results - Real Python",
             url=SEARCH_URL.format(user_search=quote_plus(user_search)),
-            description="Here are the top 5 results:",
+            description=f"Here are the top {max(5, len(articles))} results:",
```

Comment:
This could actually just be 
```suggestion
            description=f"Here are the top {len(articles)} results:",
```
as we're limiting the number returned elsewhere. This change would save us having to update the code in multiple places if we decide to change the number returned